autorest/codegen/__init__.py

```diff
@@ -135,12 +139,37 @@ def _build_package_dependency() -> Dict[str, str]:
             "dependency_msrest": "msrest>=0.6.21",
         }
 
+    @staticmethod
+    def _build_security(yaml_data: Dict[str, Any]) -> Dict[str, Any]:
+        security_yaml = yaml_data.get("security")
+        security: Dict[str, Any] = dict()
+        if security_yaml and security_yaml.get("authenticationRequired"):
+            # the key word is "AADToken" and "AzureKey"
+            aad_token = set()
+            azure_key = ""
+            for scheme in security_yaml.get("schemes"):
+                if _AAD_TOKEN == scheme["type"]:
+                    aad_token.update(scheme["scopes"])
+                elif _AZURE_KEY == scheme["type"]:
+                    # only accept the last one
+                    azure_key = scheme["headerName"]
+            if aad_token:
+                security[_AAD_TOKEN] = list(aad_token)
```

Comment:
nit: do you need to cast it back to a list?

Reply:
Yes, `BearerTokenCredentialPolicy` needs `List[str]` as input params